docs/paper/getting-started/getting-plugins.md

```diff
@@ -0,0 +1,120 @@
+---
+toc_max_heading_level: 4
+---
+
+# Getting Plugins
+
+Plugins are the way to extend the functionality of Paper, beyond the configuration files, and one of
```

Comment:
The inserted "beyond the configuration files" with commas reads a bit weird. I'd probably just remove the commas